Header header
sensor_msgs/Imu imu
sensor_msgs/MagneticField mag_field
string IMU_string
float64 yaw
float64 pitch
float64 roll
int64 range
float64 sensor_x
float64 sensor_y